EngineLayer/CompactPeptideWithModifiedMass.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Proteomics.Fragmentation;
+using Proteomics.ProteolyticDigestion;
+
+namespace EngineLayer
+{
+    //[Serializable]
```

Comment:
Zach, do we need this block of code for anything?

Reply:
No, delete it.